hack/lib/common.bash

```diff
@@ -1,26 +1,11 @@
 #!/usr/bin/env bash
 
-function array.contains {
```

Comment:
I'm using that in eventing upgrade tests. Please don't remove it.

Reply:
/hold

Reply:
Git would've just given it back to you anyway :shrug:. Pushed.

Reply:
Sure. Just need to rebase and wait for tests. Thanks.